test/models/CompanyUser/Model.test.ts

```diff
@@ -0,0 +1,57 @@
+import Database from "../../../src/config/Database";
+import { CompanyUser } from "../../../src/models/CompanyUser";
+import { Company, CompanyRepository } from "../../../src/models/Company";
+import { User, UserRepository } from "../../../src/models/User";
+import { companyMockData } from "../Company/mocks";
+import { UserMocks } from "../User/mocks";
+
+const nonExistentUuid = "7f03fcfa-93a9-476b-881a-b81a7ea9dbd3";
+
+describe("CompanyUser", () => {
+  beforeAll(() => Database.setConnection());
+  beforeEach(() => Promise.all([
+    CompanyRepository.truncate(),
+    UserRepository.truncate()
+  ]));
+  afterAll(() => Database.close());
+
+  it("needs to reference a company", () =>
+    expect(CompanyUser.create()).rejects.toThrow(
+      "null value in column \"companyUuid\" violates not-null constraint"
+    )
+  );
```

Comment:
Haria in test para el repository de CompanyUser y testearia est ahi. En el test del Model testearia cosas que no requieran usar la base de datos